brayns/common/PropertyMap.cpp

```diff
@@ -212,7 +212,7 @@ void PropertyMap::merge(const PropertyMap& input)
         };
     };
 
-    const auto isCompatbileTypes = [](Property::Type t0, Property::Type t1) {
+    const auto areCompatibleTypes = [](Property::Type t0, Property::Type t1) {
```

Comment:
const auto canBePromoted = [](Property::Type from, Property::Type to)

From the name is not clear in which order the conversion is allowed.